smdebug/core/utils.py

```diff
@@ -22,6 +25,24 @@
 logger = get_logger()
 
 
+def make_numpy_array(x):
+    if isinstance(x, np.ndarray):
+        return x
+    elif np.isscalar(x):
+        return np.array([x])
+    elif isinstance(x, tuple):
+        return np.asarray(x)
```

Comment:
this originally had a dtype. add that back here?

Reply:
See comment above.